ci/codespell-ignore-words.txt

```diff
@@ -4,13 +4,18 @@ ba
 cannotation
 coo
 curvelinear
+dedented
+falsy
 flate
+hax
 hist
+inh
 inout
 ment
+nam
 nd
 oly
-sur
+resizeable
```

Comment:
Use in 

https://github.com/matplotlib/matplotlib/blob/23730b41febeb3cc80bd4733a228f0ed22e1b6d5/doc/devel/MEP/MEP09.rst?plain=1#L9

is a valid spelling, and so adding to ignore file.

`sur` is removed as added to the `codespell` dictionary in `v2.2.0`. c.f. https://github.com/codespell-project/codespell/pull/2300